bin/spice/cmd/spice.go

```diff
@@ -24,13 +24,23 @@ import (
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 	"github.com/spiceai/spiceai/bin/spice/pkg/util"
+	"github.com/spiceai/spiceai/bin/spice/pkg/version"
 )
 
 var verbosity = util.NewVerbosity()
 
 var RootCmd = &cobra.Command{
 	Use:   "spice",
 	Short: "Spice.ai CLI",
+
+	PersistentPreRun: func(cmd *cobra.Command, args []string) {
```

Comment:
This will run on all commands? (i.e. `spice chat`)

Reply:
Yes, example for spice chat

```shell
spice chat
Spice.ai OSS CLI v1.1.0-unstable-build.3ea213f7 (https://spiceai.org/docs/cli)
Using model: gpt-4o-mini
chat> 
```